src/renderers/shaders/ShaderChunk/envmap_fragment.glsl.js

```diff
@@ -3,18 +3,28 @@ export default /* glsl */`
 
 	#ifdef ENV_WORLDPOS
 
-		vec3 cameraToVertex = normalize( vWorldPosition - cameraPosition );
+		vec3 cameraToFrag;
+		
+		if ( isOrthographic ) {
+
+			cameraToFrag = normalize( vec3( -viewMatrix[0][2], -viewMatrix[1][2], -viewMatrix[2][2] ) );
+
+		}  else {
+
+			cameraToFrag = normalize( vWorldPosition - cameraPosition );
+
+		}
```

Comment:
Not tested, but may work.
```glsl
vec3 cameraToFrag = ( isOrthographic ) ? normalize( - viewMatrix[ 2 ].xyz ) : normalize( vWorldPosition - cameraPosition );
```

Reply:
No, doesn't work. `matrix[i]` is a column selector. Unfortunately, I don't think there is a row selector on GLSL.

Reply:
Oh, right. You are selecting the row, not the column.

Only the white space needs to be corrected: `- viewMatrix[ 0 ][ 2 ]`